db/primary_migrate/20230625143140_drop_irs_attempt_api_log_files.rb

```diff
@@ -0,0 +1,5 @@
+class DropIrsAttemptApiLogFiles < ActiveRecord::Migration[7.0]
+  def change
+    drop_table :irs_attempt_api_log_files
```

Comment:
can we use the reversible `drop_table` syntax? [like here](https://joshfrankel.me/blog/ensure-dropping-a-database-table-is-reversible/#:~:text=Let%E2%80%99s%20specify%20the,the%20widgets%20table.)